test/unit/aggregate_store_test.rb

```diff
@@ -18,22 +18,28 @@ def add(attr, message)
     end
   end
 
+  def elasticsearch_store
+    @elasticsearch_store ||=
+      begin
+        store = Class.new
+        store.send(:include, Aggregate::AggregateStore)
+        store.define_singleton_method(:aggregate_db_storage_type) { :elasticsearch }
+        store
+      end
+  end
+
   context "aggregate_attribute" do
     setup do
-      @store = Class.new { }
+      @store = Class.new
       @store.send(:include, Aggregate::AggregateStore)
       @store.aggregate_attribute(:name, :string)
     end
 
     should "pass aggregate_db_storage_type option to all attribute handlers if aggregate_db_storage_type is not nil" do
-      store = Class.new { }
-      store.send(:include, Aggregate::AggregateStore)
-      store.define_singleton_method(:aggregate_db_storage_type) { :elasticsearch }
+      elasticsearch_store.aggregate_attribute(:name, :string)
+      elasticsearch_store.aggregate_attribute(:number, :integer)
```

Comment:
I'm a bit confused, do we not want to use `@store`?

Reply:
As discussed in person, we'll leave the helper method as is and not update existing test patterns given that there isn't necessarily going to be much benefit from doing so.